.github/config.yml

```diff
@@ -17,7 +17,7 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/layer5io/meshery-linkerd/master/.github/welcome/Layer5-celebration.png)
+  ![Congrats!](https://github.com/meshery/meshery-tanzu-sm/blob/master/.github/welcome/Layer5-celebration.png)
   
```

Comment:
Hey @debjani1413 this is meshery-linkerd only so I think you should not change the link here. You must only change the layer5io endpoint to meshery.

Reply:
@sayantan1413 can you please provide me the exact link to which it should point???

Reply:
@sayantan1413 I've updated the link can you please check and confirm??

Reply:
@debjani1413 this will need to be a raw GitHub link, direct to the img (and only the image)